api/transaction/routes.go

```diff
@@ -33,7 +33,7 @@ type FacadeHandler interface {
 	CreateTransaction(nonce uint64, value string, receiver string, receiverUsername []byte, sender string, senderUsername []byte, gasPrice uint64,
 		gasLimit uint64, data []byte, signatureHex string, chainID string, version uint32, options uint32) (*transaction.Transaction, []byte, error)
 	ValidateTransaction(tx *transaction.Transaction) error
-	ValidateTransactionForSimulation(tx *transaction.Transaction) error
+	ValidateTransactionForSimulation(tx *transaction.Transaction, bypassSignature bool) error
```

Comment:
If I would to blame something in this PR I would blame the fact that now we have "bypass signature = true" which means "do not check the signature". I would have made the flag, query param and all variable names sound more like "checkSignature" (with default to true if not specified). In this way when we pass true I would automatically think that the signature will be checked. "bypass signature = false" will require that everyone that will read the code will have to do a mental double negation operation which will make the code a little more hard to understand.

Reply:
Query parameter `bypassSignature` was renamed, now parameter is `checkSignature` and is by default true.